conf/be.conf

```diff
@@ -29,6 +29,13 @@ webserver_port = 8040
 heartbeat_service_port = 9050
 brpc_port = 8060
 
+# HTTPS configures
+enable_https = true
+# path of certificate in PEM format.
+ssl_certificate_path = "$DORIS_HOME/conf/cert.pem"
+# path of private key in PEM format.
+ssl_private_key_path = "$DORIS_HOME/conf/key.pem"
```

Comment:
Add an https port and  forward  like fe

Reply:
After turning on https, http is still available for the same port. Brpc server can automatically distinguish https from http requests. 
But there is no global configure to disable http request, which would be supported by brpc in the future.